translations/fr.po

```diff
@@ -2,126 +2,145 @@ msgid ""
 msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 
-#: cards/multilang-menuitem-standard/component.js:27
+#: theme-components/vertical-full-page-map-alternative-verticals/alternativeverticals.hbs:31
+msgid "([[resultsCount]] result)"
+msgid_plural "([[resultsCount]] results)"
+msgstr[0] "([[resultsCount]] résultat)"
+msgstr[1] "([[resultsCount]] résultats)"
+
+#: cards/multilang-menuitem-standard/component.js:29
 msgid "[[calorieCount]] calories"
 msgstr "[[calorieCount]] calories"
 
-#: cards/multilang-menuitem-standard/component.js:23
+#: theme-components/vertical-full-page-map-alternative-verticals/alternativeverticals.hbs:3
+msgid "<em class=\"yxt-AlternativeVerticals-noResultsInfo--emphasized\">No results found</em> in [[currentVerticalLabel]]."
+msgstr "<em class=\"yxt-AlternativeVerticals-noResultsInfo--emphasized\">Aucun résultat trouvé</em> dans [[currentVerticalLabel]]."
+
+#: cards/multilang-menuitem-standard/component.js:25
 msgid "Allergens"
 msgstr "Allergènes"
 
-#: cards/multilang-job-standard/component.js:33
+#: theme-components/vertical-full-page-map-alternative-verticals/alternativeverticals.hbs:43
+msgid "Alternatively, you can <a class=\"yxt-AlternativeVerticals-universalLink\" href=\"[[universalUrl]]\"> view results across all search categories</a>."
+msgstr "Vous pouvez également <a class=\"yxt-AlternativeVerticals-universalLink\" href=\"[[universalUrl]]\">afficher les résultats de toutes les catégories de"
+" recherche</a>."
+
+#: cards/multilang-job-standard/component.js:35
 msgid "Apply Now"
-msgstr "Postuler"
+msgstr "S'inscrire"
```

Comment:
It looks like Smartling re-translated some of our existing strings? I thought they were aware of which ones that have already been translated. Regardless, I think we should keep our existing translations since we know they are good